src/test/java/bio/terra/controller/EncodeFileTest.java

```diff
@@ -130,25 +118,18 @@ public void encodeFileTest() throws Exception {
             .table("donor")
             .path("gs://" + dataRepoConfiguration.getIngestbucket() + "/encodetest/donor.json");
 
-        jsonRequest = objectMapper.writeValueAsString(ingestRequest);
-        result = mvc.perform(post(url)
-            .contentType(MediaType.APPLICATION_JSON)
-            .content(jsonRequest))
-            .andReturn();
-        response = connectedOperations.validateJobModelAndWait(result);
-
-        ingestResponse = connectedOperations.handleAsyncSuccessCase(response, IngestResponseModel.class);
-        assertThat("ingest response has no bad rows", ingestResponse.getBadRowCount(), equalTo(0L));
+        connectedOperations.ingestTableSuccess(studySummary.getId(), ingestRequest);
 
         // At this point, we have files and tabular data. Let's make a dataset!
 
-        response = connectedOperations.launchCreateDataset(studySummary, "encodefiletest-dataset.json", "");
+        MockHttpServletResponse response = connectedOperations.launchCreateDataset(
+            studySummary, "encodefiletest-dataset.json", "");
```

Comment:
some of the connected operations combine checking for success or failure in the create call and others do the create and then call something else to verify the success. I would like to advocate that the connected operations class just does the action and that maybe there is a helper in the test that verifies the result. can be done in a follow on PR - Dan M you don't need to refactor our code :)